build/DependencyVersions.props

```diff
@@ -39,7 +39,8 @@
     <SharedHostVersion>$(MicrosoftNETCoreAppPackageVersion)</SharedHostVersion>
     <HostFxrVersion>$(MicrosoftNETCoreAppPackageVersion)</HostFxrVersion>
     <AspNetCoreVersion>$(MicrosoftAspNetCoreAppPackageVersion)</AspNetCoreVersion>
-    <WinFormsAndWpfVersion>3.0.0-alpha-26823-9</WinFormsAndWpfVersion>
+    <WinFormsAndWpfVersion>3.0.0-alpha-26828-2</WinFormsAndWpfVersion>
+    <WpfTemplatesVersion>$(WinFormsAndWpfVersion)</WpfTemplatesVersion>
```

Comment:
To make ProdCon easier, we should use a PackageVersion property matching the Wpf package being inserted here, instead of creating our own one off properties.

cc @mmitche

Reply:
So are you saying we should change the property name to `MicrosoftPrivateWpfPackageVersion` or something?

Reply:
Yep, this way it can be inserted through ProdCon.